### DISCUSSION TITLE
Chart customization freely with additional information

### DISCUSSION THREAD

**Discussion by dpedwards** [Budibase/budibase#5041] [Help] [answered]
Hello, is it possible to freely customize the charts available in BB, as in the example of the bar chart below?

#1 Wanted: 
![2022-03-23 16_28_31-JavaScript star and bar rating](https://user-images.githubusercontent.com/2211088/159737082-e13990e1-5165-488a-900d-d8070eeabf23.png)

#2 BB Bar Chart present.:
![2022-03-23 16_34_51-Budibase](https://user-images.githubusercontent.com/2211088/159737107-b029ba03-9a90-47d5-a801-c027202d0d28.png)

**Comment by mjashanks** [comment, marked as answer]
unfortunately its not possible. We do plan to add the ability to create your own custom component, which would allow you to build any component.
